src/backend/base/pyproject.toml

```diff
@@ -151,6 +151,7 @@ line-length = 120
 [tool.ruff.lint]
 select = [
     "ASYNC",
+    "B904",
```

Comment:
Not enforcing all B rules atm. Only B904, the one for `raise SomeError from exc`